test/fw/ptl/lib/pbs_testlib.py

```diff
@@ -2083,12 +2083,17 @@ def decode_value(cls, value):
         of it, if it is made of alphanumeric values only, return
         it as a string, if it is of type size, i.e., with a memory
         unit such as b,kb,mb,gb then return the converted size to
-        kb without the unit
+        kb without the unit, if value is a dictionary string return
+        dictionary value
 
         :param value: attribute/resource value
         :type value: str or int
         :returns: int or float or string
         """
+        if re.search('\'{\".*\":.*', str(value)):
```

Comment:
You are assuming that the dict will have string key & value, so this will not work for a dict that has numeric key or values.